deploy/operables.nix

```diff
@@ -170,7 +171,6 @@ in
 
       ${wait-for-socket}/bin/wait-for-socket "$CARDANO_NODE_SOCKET_PATH"
 
-      export OTEL_SERVICE_NAME=marlowe-chain-indexer
```

Comment:
Let's keep the default values for these:

```suggestion
export OTEL_SERVICE_NAME="${OTEL_SERVICE_NAME:-marlowe-chain-indexer}"
```

Reply:
@jhbertra , Would that force the need to use OTEL, or would `OTEL_SERVICE_NAME` be harmless?

Reply:
@jhbertra , Would that force the need to use OTEL, or would `OTEL_SERVICE_NAME` be harmless?